project/Http4sSitePlugin.scala

```diff
@@ -0,0 +1,205 @@
+    },
+    tlSiteHeliumConfig := {
+      val latest = Http4sPlugin.latestPerMinorVersion(baseDirectory.value)
+      // helpful to render landing page when previewing locally
+      if (version.value.startsWith("1.") || !isCi.value)
+        landingPage.configure(
+          tlSiteHeliumConfig.value,
+          latest((0, 23)).toString,
+          latest((1, 0)).toString,
+        )
+      else tlSiteHeliumConfig.value
+    },
+    laikaTheme := laikaTheme.value.extend(redirects.theme),
+  )
+
+  object landingPage {
+    def configure(helium: Helium, stableVersion: String, milestoneVersion: String): Helium =
+      helium.site.landingPage(
+        logo = Some(Image.internal(Root / "images" / "http4s-logo-text-light.svg")),
+        title = None,
+        subtitle = Some("Typeful, functional, streaming HTTP for Scala"),
+        latestReleases = Seq(
+          ReleaseInfo("Latest Stable Release", stableVersion),
+          ReleaseInfo("Latest Milestone Release", milestoneVersion),
+        ),
+        license = Some("Apache 2.0"),
+        documentationLinks = projectLinks,
+        projectLinks = Nil, // TODO
+        teasers = landingPage.teasers,
+      )
+
+    val teasers: Seq[Teaser] = Seq(
+      Teaser(
+        "Typeful",
+        "http4s servers and clients share an immutable model of requests and responses. Standard headers are modeled as semantic types, and entity codecs are done by typeclass.",
+      ),
+      Teaser(
+        "Functional",
+        "The pure functional side of Scala is favored to promote composability and easy reasoning about your code. I/O is managed through cats-effect.",
+      ),
+      Teaser(
+        "Streaming",
+        "http4s is built on FS2, a streaming library that provides for processing and emitting large payloads in constant space and implementing websockets.",
+      ),
+      Teaser(
+        "Cross-platform",
+        "http4s cross-builds for Scala.js, enabling you to share code across browsers, Node.js apps, serverless functions, and JVM servers.",
+      ),
+    )
+
+    val projectLinks: Seq[TextLink] = Seq(
+      TextLink.internal(Root / "versions.md", "Versions"),
+      TextLink.internal(Root / "changelog.md", "Changelog"),
+      TextLink.internal(Root / "getting-help.md", "Getting Help"),
+      TextLink.internal(Root / "contributing.md", "Contributing"),
+      TextLink.internal(Root / "adopters.md", "Adopters"),
+      TextLink.internal(Root / "code-of-conduct.md", "Code of Conduct"),
+      TextLink.internal(Root / "further-reading.md", "Further Reading"),
+    )
+
+  }
+
+  object versions {
+    def config(isCi: Boolean): Versions = Versions(
+      currentVersion = current,
+      olderVersions = all.dropWhile(_ != current).drop(1),
+      newerVersions = all.takeWhile(_ != current),
+      // helpful to render unversioned pages when previewing locally
+      renderUnversioned = current == v1_0 || !isCi,
```

Comment:
@danicheg this `renderUnversioned` setting disables the landing page from being generated by the 0.22/0.23 branches.

Reply:
Got it, thanks ❤️